src/sentry/runner/initializer.py

```diff
@@ -79,6 +79,9 @@ def get_asset_version(settings):
     'mail.use-tls': 'EMAIL_USE_TLS',
     'mail.from': 'SERVER_EMAIL',
     'mail.subject-prefix': 'EMAIL_SUBJECT_PREFIX',
+    'mail.enable-replies': 'SENTRY_ENABLE_EMAIL_REPLIES',
+    'mail.reply-hostname': 'SENTRY_SMTP_HOSTNAME',
+    'mail.mailgun-api-key': 'MAILGUN_API_KEY',
```

Comment:
Good eye. I fixed. There was no reason they needed to be ported back into Django settings, and I moved into the legacy settings.